package-lock.json

```diff
@@ -55694,7 +55694,7 @@
     },
     "packages/rum": {
       "name": "@elastic/apm-rum",
-      "version": "5.16.0",
+      "version": "5.16.1",
```

Comment:
weird why you got this change?

Reply:
I got it just after `npm i` on a clean checkout